packages/performance-tests/test/cases/TestCaseContext.ts

```diff
@@ -0,0 +1,8 @@
+import { BriefcaseDb } from "@itwin/core-backend";
+import { TestTransformerModule } from "../TestTransformerNodule";
+
+export type TestCaseContext = {
+    sourceDb: BriefcaseDb;
+    transformerModule: TestTransformerModule;
+    addReport: (...smallReportSubset: [testName: string, iModelName: string, valDescription: string, value: number]) => void;
```

Comment:
I didn't like this original type much a while ago but now that we're moving things around again, extract this complicated type to like `ReportCallback` without the redundant spread-tuple:
```ts
type ReportCallback =  (testName: string, iModelName: string, valDescription: string, value: number) => void;
```